packages/server/medplum.config.json

```diff
@@ -21,6 +21,7 @@
   "defaultBotRuntimeVersion": "vmcontext",
   "allowedOrigins": "*",
   "introspectionEnabled": true,
+  "mcpEnabled": true,
```

Comment:
Just checking that this is intentionally being enabled by default for the dev config file.

Reply:
I debated that.  I'm ok to leave it out for now.